tests/functional/ext/magic_comparison/magic_comparison.py

```diff
@@ -0,0 +1,23 @@
+"""
+Checks that magic numbers are not used in comparisons
+"""
+# pylint: disable=missing-docstring,invalid-name,too-few-public-methods,import-error,wrong-import-position, comparison-of-constants, singleton-comparison
```

Comment:
```suggestion
# pylint: disable=missing-docstring,invalid-name,too-few-public-methods,import-error,wrong-import-position
```
This would help make pylint behavior clearer when reading the functional test, right ?

Reply:
probably, and add the messages in the `.txt` file? even if it is not my checker?